core/org.wso2.carbon.core/src/main/java/org/wso2/carbon/core/util/KeyStoreManager.java

```diff
@@ -463,12 +347,14 @@ public Certificate getCertificate(String keyStoreName, String alias) throws Exce
     }
 
     /**
+     * @deprecated
```

Comment:
Add a deprecated comment. Why was it deprecated / What to use instead?

Reply:
Fixed in the latest changes